test/test_mapper_llvm.cc

```diff
@@ -66,6 +65,176 @@ def fun(float(N, M) A, float(N, M) B) -> (C) {
   checkRtol(Cc - C, {A, B}, N * M);
 }
 
+TEST(LLVMCodegen, BasicParallel) {
+  string tc = R"TC(
+def fun(float(N, M) A, float(N, M) B) -> (C) {
+  C(i, j) = A(i, j) + B(i, j)
+}
+)TC";
+  auto N = 40;
+  auto M = 24;
+
+  auto ctx = isl::with_exceptions::globalIslCtx();
+  auto scop = polyhedral::Scop::makeScop(ctx, tc);
+  auto context = scop->makeContext(
+      std::unordered_map<std::string, int>{{"N", N}, {"M", M}});
+  scop = Scop::makeSpecializedScop(*scop, context);
+  scop =
+      Scop::makeScheduled(*scop, SchedulerOptionsView(SchedulerOptionsProto()));
```

Comment:
Let's please rebase on top of #197, for this case you should create a CpuMappingOptions 
I am adding that in a followup because #197 is big enough and is already having enough merge conflicts as it is.

Reply:
Also, why not:
```
SchedulerOptionsProto p;
SchedulerOptionsProtoView pv(p);
scop = Scop::makeScheduled(*scop, pv);
```
?